src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -311,7 +311,7 @@ public static JSONObject prepareUpload(final byte[] documentContents) throws Uns
 
     private String performPostRequest(HttpURLConnection conn, JSONObject body, int expectedStatusCode) throws NotificationClientException {
         try{
-            OutputStreamWriter wr = new OutputStreamWriter(conn.getOutputStream());
+            OutputStreamWriter wr = new OutputStreamWriter(conn.getOutputStream(), "utf-8");
```

Comment:
Is it better to use `StandardCharsets.UTF_8` here? Available since Java 7.

Reply:
Sure I can make that change

Reply:
Thank you - it's been awhile since I've written java